src/main/java/org/citeplag/controller/BaseXController.java

```diff
@@ -66,14 +68,58 @@ public MathRequest xQueryProcessing(
         return process(query, "xquery", request);
     }
 
+
+    @PostMapping("/mwsquery_old")
+    @ApiOperation(value = "Run MWS query on BaseX")
+    public MathRequest mwsProcessingOld(@RequestParam String query, HttpServletRequest request) {
```

Comment:
param could not be resolved since the request from formulasearch is 'encrypted' by url-encoding.